lib/rspec-puppet-facts.rb

```diff
@@ -298,9 +294,7 @@ def self.mcollective?
   # @return [Array<Hash>]
   # @api private
   def self.meta_supported_os
-    unless metadata['operatingsystem_support'].is_a? Array
-      fail StandardError, 'Unknown operatingsystem support in the metadata file!'
-    end
+    raise StandardError, 'Unknown operatingsystem support in the metadata file!' unless metadata['operatingsystem_support'].is_a? Array
```

Comment:
I'd say this is less readable. Can we disable the guard condition cop?

Reply:
https://github.com/voxpupuli/rspec-puppet-facts/pull/184